### PR TITLE
Setup reporting when binding and make report feature deprecated

### PR DESCRIPTION
**Pull request by Koenkk**
Docs: https://github.com/Koenkk/zigbee2mqtt.io/pull/570

### CODE REVIEW

File: lib/extension/bind.js [hunk 1]
```diff
@@ -5,16 +5,115 @@ const legacyTopicRegex = new RegExp(`^${settings.get().mqtt.base_topic}/bridge/(
...
+    ],
+    'closuresWindowCovering': [
+        {attribute: 'currentPositionLiftPercentage', ...defaultReportConfiguration},
+        {attribute: 'currentPositionTiltPercentage', ...defaultReportConfiguration},
+    ],
+};
+
+const pollOnMessage = [
+    {
+        // Key is used this.pollDebouncers uniqueness
```
Comment (nurikk) [inline lib/extension/bind.js:61]: Why not use the array index as a key?

Comment (Koenkk) [reply lib/extension/bind.js]: Good comment, added, thanks!

**Comment by muddy79** [comment]
May I ask: what will happen after the reporting feature has been eventually removed? Do I need to re-bind ALL my control devices to the proper groups to keep the reporting enabled? It's quite a huge work in my case (~20 devices), considering the fact that the remote control devices can't be in a sleep state while binding.... That doesn't sound enjoyable.

**Comment by Koenkk** [comment]
Check second point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.17.1 (TLDR; nothing will change if you leave it as is)

**Pull request merged**